src/components/page/Home.js

```diff
@@ -31,9 +31,14 @@ const useStyles = makeStyles({
     },
     graphComponentRow: {
         display: "grid",
-        gridTemplateColumns: "repeat(auto-fit, minmax(680px, 1fr))",
+        [theme.breakpoints.between("sm", "md")]: {
+            gridTemplateColumns: "repeat(auto-fit, 1fr)",
+        },
+        "@media (min-width: 1280px)": {
+            gridTemplateColumns: "repeat(auto-fit, minmax(554px, 1fr))",
+        },
```

Comment:
Grid improvements for graphs, on larger screens it will try to fit as many of them as it can on a single row. On smaller screens it will have 1 graph per row.